includes/class-sensei-messages.php

```diff
@@ -551,6 +552,20 @@ public function message_login() {
 			}
 		}
 	}
+
+	/**
+	 * Exclude message comments from feed queries and RSS.
+	 *
+	 * @since 2.1.0
+	 * @access private
+	 *
+	 * @param  string $where The WHERE clause of the query.
+	 * @return string
+	 */
+	public static function exclude_message_comments_from_feed_where( $where ) {
```

Comment:
I think this function should not be static?

Reply:
Good catch! Fixed and rebased.